### PR TITLE
Add service proivder to module builder 

### PR DESCRIPTION
**Pull request by AmrAhmedFekry**
Make module more stand-alone form project, add 
Add some features to generated postman file

### CODE REVIEW

File: cloneable-modules/users/Controllers/Admin/UsersController.php [hunk 1]
```diff
@@ -12,6 +13,59 @@ class UsersController extends AdminApiController
      */
     protected $controllerInfo = [
         'repository' => 'users',
-        'listOptions' => [],
+        'listOptions' => [
+            'select' => ['id','name','user_group_id','email'],
```
Comment (hassanzohdy) [inline cloneable-modules/users/Controllers/Admin/UsersController.php:17]: Set it empty to allow the developer to determine what to select by him/herself.

Comment (hassanzohdy) [inline cloneable-modules/users/Controllers/Admin/UsersController.php:17]: Set it empty to allow the developer to determine what to select by him/herself.

File: cloneable-modules/users/Controllers/Admin/UsersController.php [hunk 2]
```diff
@@ -12,6 +13,59 @@ class UsersController extends AdminApiController
...
+            'filterBy' => [],
+            'paginate' => null, // if set null, it will be automated based on repository configuration option
+        ],
+        'rules' => [
+            'all' => [
```
Comment (hassanzohdy) [inline cloneable-modules/users/Controllers/Admin/UsersController.php:22]: Set it empty to allow the developer to determine what to validate by him/herself.

File: cloneable-modules/users/Controllers/Admin/UsersController.php (hunk 1 above)
Comment (AmrAhmedFekry) [reply cloneable-modules/users/Controllers/Admin/UsersController.php:17]: Do you think it will be good to fill it as default ?

**Pull request merged**